src/platform/silabs/wifi/lwip-support/dhcp_client.h

```diff
@@ -31,6 +31,24 @@ extern "C" {
 #define DHCP_TIMEOUT ((uint8_t) 4)
 #define DHCP_LINK_DOWN ((uint8_t) 5)
 
+/************************** Station Static Default ****************************/
+#define STA_IP_ADDR0_DEFAULT ((uint8_t) 192) ///< Static IP: IP address value 0
```

Comment:
where are these used?

The comments seem odd: what is value 0/1/2/3 ? It seems to be the byte value, but then again why do we need it like this? I do not see these used in the PR.

Reply:
Yeah, this was unused code, which was added removed the unused code.